three.c

```diff
@@ -0,0 +1,90 @@
+#include <stdio.h>
+#include <math.h>      //This program determines whether three numbers could form a tringle and what kind of triangle they would form
+#include <stdlib.h>
+
+int isTriangle(int a, int b, int c)
+{
+	if (a+b<c)
+	{
+		printf("This is not a valid triangle, restart the program\n");
+	}
+	if (a+c<b)
+	{
+		printf("This is not a valid triangle, restart the program\n");
+	}
+	if (b+c<a)
+	{
+		printf("This is not a valid triangle, restart the program\n");
+	}
+	else
```

Comment:
this prints sometimes when it shouldn't. The reason for this is you kept these as all if statements instead of ` else if `